WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -317,83 +258,11 @@ final class OrderListViewModel {
                                  pageSize: pageSize,
                                  reason: reason,
                                  lastFullSyncTimestamp: lastFullSyncTimestamp,
-                                 completionHandler: { [weak self] timeInterval, error in
-            /// A bit of a side-effect: `onDidChangeContent` is not called for first load
-            self?.ippSurveySource = self?.feedbackBannerSurveySource()
+                                 completionHandler: { timeInterval, error in
             completionHandler(timeInterval, error)
         })
     }
 
-    private func setupWCPayIPPResultsControllers() {
-        let updateFeedbackSurveySource = { [weak self] in
-            guard let self = self else { return }
-            self.ippSurveySource = self.feedbackBannerSurveySource()
-        }
-        wcPayIPPOrdersResultsController.onDidChangeContent = updateFeedbackSurveySource
-        recentWCPayIPPResultsController.onDidChangeContent = updateFeedbackSurveySource
-
-        ippSurveySource = feedbackBannerSurveySource()
-    }
-
-    private func fetchIPPTransactions() {
-        do {
-            try wcPayIPPOrdersResultsController.performFetch()
-            try recentWCPayIPPResultsController.performFetch()
-        } catch {
-            DDLogError("Error fetching IPP transactions: \(error)")
-        }
-    }
-
-    func trackInPersonPaymentsFeedbackBannerShown(for surveySource: SurveyViewController.Source?) {
-        var campaign: FeatureAnnouncementCampaign? = nil
-
-        switch surveySource {
-        case .inPersonPaymentsCashOnDelivery:
-            campaign = .inPersonPaymentsCashOnDelivery
-        case .inPersonPaymentsFirstTransaction:
-            campaign = .inPersonPaymentsFirstTransaction
-        case .inPersonPaymentsPowerUsers:
-            campaign = .inPersonPaymentsPowerUsers
```

Comment:
I think we can delete these three cases from `FeatureAnnouncementCampaign` as well